app/webpacker/components/CompetitionForm/FormSections/RegistrationFees.js

```diff
@@ -49,10 +49,10 @@ export default function RegistrationFees() {
         />
       )}
       <ConditionalSection showIf={canRegOnSite}>
-        <InputCurrencyAmount id="onTheSpotEntryFee" currency={currency} required={canRegOnSite} />
+        <InputCurrencyAmount id="onTheSpotEntryFee" currency={currency} required={canRegOnSite} ignoreDisabled />
```

Comment:
Should this be `ignoreDisabled` per WCAT's spec?

Reply:
It has to be. Turning OTS on/off doesn't make sense without being able to edit the fee.